GWAS/LMM.ipynb

```diff
@@ -1139,6 +1140,43 @@
     "      gzip -f --best ${_output:n}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "[regenie_burden_3]\n",
+    "# Select the annotations to be used in the mask file. format: mask# annotatio type\n",
+    "parameter: mask_file = path(\".\")\n",
+    "# Select the upper MAF to generate masks\n",
+    "parameter: aaf_bins =[0.05]\n",
+    "f = open(mask_file, \"r\")\n",
+    "masks = [i.split(\" \")[0] for i in f.readlines()]\n",
+    "output: [f'{cwd}/cache/{_input:bn}_'+ str(phenoCol[i]) + \"_\" + str(masks[j]) + \".\" + str(aaf_bins[k]) + '.regenie.gz' for i in range(len(phenoCol)) for j in range(len(masks)) for k in range(len(aaf_bins))] \n",
+    "task: trunk_workers = 1, trunk_size = job_size, walltime = '12h', mem = '15G', cores = numThreads, tags = f'{step_name}_{_output[0]:bn}'\n",
+    "python: container=container_lmm, expand = \"${ }\", stderr = f'{cwd}/cache/{_input:bn}.stderr', stdout = f'{cwd}/cache/{_input:bn}.stdout', volumes = [f\"{cwd:a}:{cwd:a}\"]\n",
+    "    import pandas as pd \n",
+    "\n",
+    "    if len(${phenoCol}) == 1:\n",
+    "        _input = [\"${_input}\"]\n",
+    "    else:\n",
+    "        _input = [f'{i}' for i in \"${_input}\".split(\" \")]\n",
+    "\n",
+    "    allele_combos = [str(m) + \".\" + str(a) for m in ${masks} for a in ${aaf_bins}]\n",
```

Comment:
@tfabiha I see you are manually creating these combos. That'd work. The mask will will be the same as used for regenie burden test?

Reply:
@gaow yes, we decided to do it this way because doing it from the yml file seemed more complicated